gui/src/components/settings/pages/GeneralSettings.tsx

```diff
@@ -184,6 +193,10 @@ export function GeneralSettings() {
     unitDisplay: 'narrow',
     maximumFractionDigits: 2,
   });
+  const degreeFormat = new Intl.NumberFormat(currentLocales, {
+    style: 'unit',
+    unit: 'degree',
+  });
```

Comment:
We should maybe use our translation framework? Idk about this

Reply:
@ImUrX Can I get your feedback on this 🐱

Reply:
its fine, i discovered a better integrated way on Fluent recently but for now this is how we been doing it, and if its per second you can do `degree-per-second` for the `unit`